guardian-ui/README.md

```diff
@@ -0,0 +1,32 @@
+# Guardian UI
+
+## Prerequisites NPM and Yarn
+
+(NPM)[https://www.npmjs.com/] - (use [nvm](https://github.com/nvm-sh/nvm))
+
+(Yarn)[https://yarnpkg.com/getting-started/installhttps://classic.yarnpkg.com/lang/en/docs/install/] - `npm install --global yarn`
```

Comment:
we should prefer yarn installed in nix shell. See this reference https://github.com/fedimint/fedimint/blob/master/docs/gateway.md#developing-gateway-ui-aka-mintgate

Reply:
Should this doc be located with the other ones then?

Reply:
Having both is not a problem. But the guide to enter `fedimint-ui` nix dev shell should be the same

Reply:
I'm going to wait to update `ui.md` once the new UI is finished. Then can update with specific instructions.